google-cloud-language/lib/google/cloud/language/document.rb

```diff
@@ -180,7 +180,8 @@ def language= new_language
         #   Optional. The default is `false`. If every feature option is
         #   `false`, **all** features will be performed.
         # @param [String] encoding The encoding type used by the API to
```

Comment:
The param type is `[String]`, but the acceptable values listed are symbols. Want to add `Symbol` to the types lists, or remove the colon from the acceptable values to make them strings? (I vote for the strings, since they are upper case).